packages/@apphosting/adapter-nextjs/src/interfaces.ts

```diff
@@ -76,6 +76,7 @@ export interface RoutesManifest {
 export interface OutputBundleOptions {
   bundleYamlPath: string;
   outputDirectory: string;
+  outputBundleAppDir: string;
```

Comment:
I think it's time to document this interface :) Can you add jsdocs for each field to explain what they are? Can you also do an audit to make sure the field names are as unambiguous as possible?

Reply:
Added docs and renamed some variables to make things more clear